### ISSUE TITLE
Wand Attack into Intervene buff

### ISSUE THREAD

**Issue by Evolvee** [ccshiro/cc-buglist#2228]
How this works atm:
Warrior intervenes a target1, target 2 shoots a wand shot into target1, target 1 gets combat, takes dmg from wand shot and also remove the Intervene buff...

How this in my opinion should work:
Warrior intervenes a target1, target 2 shoots a wand shot into target1, target 1 takes no dmg, Warrior takes it instead and Intervene buff is removed.

**Comment by Stoney21** [comment]
Can't find anything to confirm this. Wand damage isn't physical damage (even though the color of its damage is white) and Intervene buff works only on physical damage.

http://wowwiki.wikia.com/wiki/Physical

It probably shouldn't do anything to Intervene buff.

**Comment by Schaka** [comment]
As far as I remember on retail, it shouldn't take the buff and just damage the target.
But since it's a physical in the same way that arcane shot is, I think my memory might be wrong.

**Comment by Evolvee** [comment]
well it was used by PR combs against WD combs to remove the "pre-intervene" which prevented rogue to open on druid when he came out of stealth(or forced him to use vanish on it), but im not sure if it should dmg the target or the warrior who used intervene(thats why i said in my opinion) :/

>  Intervene buff works only on physical damage.

cheap shot isnt damage as well

but we can probably agree that it shouldnt work how it works atm

**Comment by Schaka** [comment]
Agreed. There was also a range to intervene on retail, where when the warrior ran too far from its target, something like cheap shot wouldn't be redirected. I've noticed that quite a few times playing War/Druid on retail with my brother, where we had no clue how to bear mage/rogue and tried playing around with intervene. Although it did feel like the range wasn't that of the originally intervened spell.

**Comment by DarkenRal** [comment]
> cheap shot isnt damage as well

But it's a Physical ability. Physical is not only damage, Physical is classified as a "school of magic".

> well it was used by PR combs against WD combs to remove the "pre-intervene" which prevented rogue to open on druid when he came out of stealth(or forced him to use vanish on it)

Link some video or some PR guide where you can read it or at least something to prove this. I think Schaka is right, it should not interact with Intervene in any way (neither redirect damage nor consume buff).

**Comment by Evolvee** [comment]
i was reacting on physical dmg, not on physical ability

**Comment by DarkenRal** [comment]
oh, i thought you mean it should not be redirected. well, actually Stoney21 meant that wand is not physical damage and Intervene redirects only physical damage (it's not [only damage], but [only physical]) and thus wand attack can't be redirected by Intervene.

**Comment by Evolvee** [comment]
https://github.com/FeenixServerProject/Archangel_2.4.3_Bugtracker/issues/2672
something from peenix, but how can u trust peenix :D
